### ISSUE TITLE
create a planform calculation for "surface deposit age"

### ISSUE THREAD

**Issue by amoodie** [DeltaRCM/DeltaMetrics#21]
something like:
```
age = np.argmin(_strata < _strata[-1, ...], axis=0)

fig, ax = plt.subplots()
im = ax.imshow(age)
dm.plot.append_colorbar(age, ax)
plt.show()
```
![image](https://user-images.githubusercontent.com/8801322/91186176-5141f400-e6b4-11ea-8ea4-507be36a7281.png)

with additional conversions to shift the indices into ages based on the `time` attribute.
Note `_strata` is the 3d output of stratigraphy surfaces.

This would be a *function* in `plan.py` that might have a wrapper method in the `Cube` *classes*. Function should take either a `Cube` or 3d array, and return a 3d array or optionally with `return_plan=True` a `PlanformVariable` object.

**Comment by amoodie** [comment]
because my note on `_strata` above is not super clear:

```python
_strata = any_StratigraphyCube.strata
```

**Comment by amoodie** [comment]
closed with #96
